### ISSUE TITLE
raise an error if input file is not in specific suffix

### ISSUE THREAD

**Issue by ckxz105** [usc-isi-i2/t2wml#122]
raise error if input file is not in specific suffix.
For some functions, e.g. `t2wml.api.add_properties_from_file`, the function only check if some specific suffix in the file name, but it does not raise the error if it did not fit those suffix.

i.e.
```
def add_properties_from_file(file_path):
    if Path(file_path).suffix == ".json":
        with open(file_path, 'r') as f:
            input_dict= json.load(f)
    if Path(file_path).suffix == ".tsv":     
        with open(file_path, 'r') as f:
            reader=csv.DictReader(f, delimiter="\t")
            input_dict={row_dict["node1"]: str(row_dict["node2"]) for row_dict in reader if row_dict["label"]=="data_type"}
```

If I run send `file_path=="abcde"`, the function will continue until it raise some strange errors and it makes people hard to debug. We should raise an error saying "Can't recognize the file input format" once we discover this but not continue to run like:
```
    if Path(file_path).suffix == ".json":
        with open(file_path, 'r') as f:
            input_dict= json.load(f)
    if Path(file_path).suffix == ".tsv":     
        with open(file_path, 'r') as f:
            reader=csv.DictReader(f, delimiter="\t")
            input_dict={row_dict["node1"]: str(row_dict["node2"]) for row_dict in reader if row_dict["label"]=="data_type"}
    else:
        # raise some errors here
`
```

**Comment by devowit** [comment]
do you have other examples of this happening aside from properties file? I've just fixed that (in ec9a9313b) and looked around for more examples but did not find.

**Comment by devowit** [comment]
tentatively closed unless @ckxz105  has other examples
